### DISCUSSION TITLE
Um.. what's that blue dot?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22253] [New to GitHub]
<p>So there’s a blue dot on the corner of my profile pic and it bothers me… what does it mean? This started like an hour ago. :thinking:</p>
<p><img src="https://user-images.githubusercontent.com/10137/181069741-82f42a55-9dc3-4289-bbc4-f925ccf17b6b.png" alt="Screenshot 2020-11-16 at 7.32.52 PM" data-base62-sha1="zLR3ycPmXSpnbX0AeoltaBl7pQY" width="165" height="58"></p>
<p>Sorry if this is a stupid question, I just want to know what it means in case it’s important. :sweat_smile: Thanks for your time.</p>
<p>Edit: Apparently, this is happening to a lot of people.</p>

**Comment by hammerhai** [comment, marked as answer]
<p>It is used when there is a new feature available that users can preview.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/2/a27a9476229cf121e53a17d6d53fc5dc67c96ded.jpeg" data-download-href="/uploads/short-url/nblYxEr44j8H25peD2kej3SWcjP.jpeg?dl=1" title="FeaturePreview" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/43486313/181069742-148796f0-927d-446f-972a-42f182e87dcc.jpeg" alt="FeaturePreview" data-base62-sha1="nblYxEr44j8H25peD2kej3SWcjP" width="689" height="349" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a27a9476229cf121e53a17d6d53fc5dc67c96ded_2_689x349.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a27a9476229cf121e53a17d6d53fc5dc67c96ded_2_1033x523.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/2/a27a9476229cf121e53a17d6d53fc5dc67c96ded.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a27a9476229cf121e53a17d6d53fc5dc67c96ded_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">FeaturePreview</span><span class="informations">1148×581 117 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<small>“Improved container support” is a new feature, which is why you see the blue dot! “Feature preview” will lead to the blue dot. Clicking on “Feature preview” opens a dialog.<small></small></small>

**Comment by Jasper-M** [comment]
<p>Consider propagating the blue dot to the <code>Feature preview</code> option.</p>

**Comment by maxnoe** [comment]
<p>Yes, please show where this notification is actually coming from and give a way to acknowledge the message / hide the blue dot other than by enabling the beta feature.</p>

**Comment by Kestrer** [comment]
<p>Clicking away from the feature preview popup window will get rid of the blue dot without enabling the feature.</p>

**Comment by LaysDragon** [comment]
<p>Oh wow, I have the same question while I noticed this blue dot. I thought it just a decoration since I not figure out what it is too. thanks for answer.</p>

**Comment by johndevs** [comment]
<p>I thought it meant I was “Online” :grin: Thanks for the answer!</p>
